app/src/main/contrib/components/Responsibilities/ModelRunParameters/UploadModelRunParametersForm.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import {UploadFileForm} from "../../../../shared/components/UploadFileForm";
+
+interface ModelRunParametersUploadFormProps {
+    groupId: string;
+    touchstoneId: string;
+    diseases: string[];
+    token: string;
+}
+
+interface ModelRunParametersUploadFormState {
+    diseaseSelected: boolean;
+    descriptionSelected: boolean;
+}
+
+export class UploadModelRunParametersForm extends React.Component<ModelRunParametersUploadFormProps, ModelRunParametersUploadFormState> {
```

Comment:
Because we're not submitting via js... I haven't looked into it, but I don't know how much it has to add when using a native form post? I guess the only thing would be field validation. We aren't using it right now for uploading burden estimates.

Reply:
Of course, the model run paramater set files are not so big, so maybe we shouldn't be using the native form post at all for this...